src/SConscript

```diff
@@ -12,7 +12,8 @@ try:
 except:
     # If SCons is using an old python (2.4 is relatively common still), then the above may fail.
```

Comment:
Good point.  The `nada()` call looks like maybe a way to force the except block for testing.  `git blame` says it was me that added it, so probably I just forgot to remove it after testing that the except block was working properly.  Oops.  Did now.  (Of course, the "2.4 is relatively common still" comment is indeed out of date.  None of my systems have that anymore.)